LibTrixi.jl/src/api_jl.jl

```diff
@@ -49,3 +49,53 @@ function trixi_step_jl(simstate)
 
     return nothing
 end
+
+function trixi_ndims_jl(simstate)
+    mesh, _, _, _ = Trixi.mesh_equations_solver_cache(simstate.semi)
```

Comment:
Instead of using `Trixi.xxx` here and elsewhere, you could just add all required symbols to the `using Trixi: ..., xxx` statement in `src/LibTrixi.jl`. But I'll leave this up to you (I'd just prefer something consistent, such that one does not have to think - either *always* add them to the `using` statement, or *never* add them and use the qualified access everywhere, as you do here)